File: JUnitProject/src/test/java/employee/EmployeeTests.java
package employee;

import base.BaseTest;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.params.ParameterizedTest;
import org.junit.jupiter.params.provider.ValueSource;
import pages.EmployeePage;
import pages.LoginPage;

public class EmployeeTests extends BaseTest {

    public EmployeeTests(){
        System.out.println("EmployeeTests Thread-id: " + Thread.currentThread().getId());
    }

    @ParameterizedTest
    @ValueSource(strings = { "Juan", "Pedro" })
    public void testAddEmployee(String name){
        System.out.println("testAddEmployee Thread-id: " + Thread.currentThread().getId());
        System.out.println("testAddEmployee Thread-id: " + testVar);

        LoginPage loginPage = new LoginPage(webDriver);
        EmployeePage employeePage = loginPage.loginAs("admin","admin123");
        employeePage.addEmployee(name,"Juan@gmail.com","MTZ","522255",
                "Montevideo", "Uruguay", "11523");
        Assertions.assertTrue(employeePage.isSuccessAlertVisible());
    }
}

File: NUnitProject/NUnitProject/Tests/EmployeeTests.cs
using NUnit.Framework;
using NUnitProject.Pages;
using NUnitProject.Tests.Base;
using System;
using System.Collections.Generic;
using System.Diagnostics;
using System.Threading;

namespace NUnitProject.Tests
{
    public class EmployeeTests : BaseTest
    {
        public EmployeeTests()
        {
            Logger.Instance.Info("EmployeeTests Thread-id: " + Thread.CurrentThread.ManagedThreadId);
        }

        [TestCaseSource("DataProviderEmployee")]
        public void TestAddEmployee(String name)
        {
            Logger.Instance.Info("testAddEmployee Thread-id: " + Thread.CurrentThread.ManagedThreadId);

            LoginPage loginPage = new LoginPage(WebDriver);
            EmployeePage employeePage = loginPage.LoginAs("admin", "admin123");
            employeePage.AddEmployee(name, "Juan@gmail.com", "MTZ", "522255",
                    "Montevideo", "Uruguay", "11523");
            Assert.IsTrue(employeePage.IsSuccessAlertVisible());
        }

        static IEnumerable<string> DataProviderEmployee()
        {
            Logger.Instance.Info("testAddEmployee data-provider Thread-id: " + Thread.CurrentThread.ManagedThreadId);
            yield return "Juan";
            yield return "Pedro";
        }
    }
}
